submarine-workbench/workbench-web-ng/src/app/pages/workbench/workbench.component.html

```diff
@@ -71,7 +71,7 @@ <h1>Submarine</h1>
           class="operation-user-info"
         >
           <nz-avatar nzIcon="user" nzSize="small"></nz-avatar>
-          {{ userInfo.name }}
+          admin
```

Comment:
No! `userInfo.name` This username is queried from the database and cannot be written as a fixed value.